src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -184,44 +173,34 @@ protected AbstractAuditLog(Settings settings, final ThreadPool threadPool, final
             log.info("Configured Users to ignore for write compliance events: {}", ignoredComplianceUsersForWrite);
         }
 
-
-
         ignoreAuditRequests = settings.getAsList(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_IGNORE_REQUESTS, Collections.emptyList());
         if (ignoreAuditRequests.size() > 0) {
             log.info("Configured Requests to ignore: {}", ignoreAuditRequests);
         }
 
-        // check if some categories are invalid
-        for (String event : disabledRestCategories) {
-        	try {
-        		AuditMessage.Category.valueOf(event.toUpperCase());
-        	} catch(Exception iae) {
-        		log.error("Unkown category {}, please check opendistro_security.audit.config.disabled_categories settings", event);
-        	}
-		}
+        this.excludeSensitiveHeaders = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_EXCLUDE_SENSITIVE_HEADERS, true);
+    }
 
-        // check if some categories are invalid
-        for (String event : disabledTransportCategories) {
-            try {
-                AuditMessage.Category.valueOf(event.toUpperCase());
-            } catch(Exception iae) {
-                log.error("Unkown category {}, please check opendistro_security.audit.config.disabled_categories settings", event);
-            }
+    private static List<String> getConfigList(final Settings settings,
+                                              final String key,
+                                              final List<String> defaultList) {
+        List<String> list = settings.getAsList(key, defaultList);
+        if (list.size() == 1 && "NONE".equals(list.get(0))) {
```

Comment:
I'd recommend start deprecating `"NONE"` as a setting and log a warning. It is easy to use "none" instead of "NONE" especially that category names are case-insensitive.

Reply:
Then it should complain and throw exception. It is documented to use NONE https://opendistro.github.io/for-elasticsearch-docs/docs/security-audit-logs/#exclude-categories

Reply:
OK, I'll file an issue and see what others think.